docs/dapps/smart-contracts/using-hardhat-with-the-avalanche-c-chain.md

```diff
@@ -48,6 +48,12 @@ Hardhat uses `hardhat.config.js` as the configuration file. You can define tasks
 
 In our repository we use a pre-configured file [hardhat.config.ts](https://github.com/ava-labs/avalanche-smart-contract-quickstart/blob/main/hardhat.config.ts). This file configures necessary network information to provide smooth interaction with Avalanche. There are also some pre-defined private keys for testing on a local test network.
 
+:::info
+
+The port in this tutorial uses 9650. Depending on how you start your local network, it could be different. Please check [here](../../quickstart/create-a-local-test-network.md#retrieve-all-nodes) to see how to retrieve the port numbers.
```

Comment:
Are there references to port 9650 in this article? We can remove this paragraph if none.

Reply:
I would say yes because the QuickStart-Smart-Contract repo we reference uses 9650